src/Orleans.Core.Abstractions/Core/IGrainFactory.cs

```diff
@@ -166,17 +55,6 @@ public interface IGrainFactory
         /// </returns>
         TGrainInterface GetGrain<TGrainInterface>(GrainId grainId) where TGrainInterface : IAddressable;
```

Comment:
I think this one could also be removed since we have `IAddressable GetGrain(GrainId grainId, GrainInterfaceType interfaceType);`